### DISCUSSION TITLE
How to connect to remotely hosted sqlite database?

### DISCUSSION THREAD

**Discussion by nooblyf** [prisma/prisma#18852] [Q&A]
I have a SvelteKit application which i want to connect it to my sqlite database, in the docs i can only find how to connect to a local sqlite database file, is there a way to connect to sqlite database file through a url hosted on a s3 bucket?

**Comment by ludralph** [comment]
Hi @nooblyf 

SvelteKit's SQLite adapter currently only supports connecting to local SQLite files, so you will need to use a third-party library to download the file from S3 and save it locally in order to connect to it.

**Reply by nooblyf** [reply]
I'm actually using it with Prisma so i thought there would be a way through that only, but because it is a serverless environment saving a file locally won't work in production

**Reply by ludralph** [reply]
Hi @nooblyf, this [article](https://kisaragi-hiu.com/kemdict-sveltekit-sqlite) shows how you can connect to sqlite in a serverless environment

**Reply by ludralph** [reply]
hi @nooblyf 

do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
